app/components/wallet/WalletReceive.scss

```diff
@@ -89,18 +92,20 @@
     line-height: 22px;
 
     h2 {
+      font-size: 18px;
       font-family: var(--font-medium);
-      font-size: 11px;
+      font-weight: 500;
```

Comment:
This seems redundant because we already have the `font-family`. There are cases that is okay, but those are when the fonts are loaded asynchronically.